util/kube/kube.go

```diff
@@ -457,3 +457,11 @@ func WatchWithRetry(ctx context.Context, getWatch func() (watch.Interface, error
 	}()
 	return ch
 }
+
+func GetDeploymentReplicas(u *unstructured.Unstructured) *int64 {
```

Comment:
minor - a test might be nice

Reply:
yep, I will add, I see that coverage is down